+dj/Computed.m

```diff
@@ -1,5 +1,10 @@
 classdef Computed < dj.internal.AutoPopulate
     % defines a computed table
+    properties(Constant)
+        tierRegexp = ['(?<computed>' ...
```

Comment:
I think `sprintf` would be clearer than these types of concatenations.

Reply:
@dimitri-yatsenko Updated styling per your feedback.